files-community/Files
# Feature: Add edit tags button to the preview pane

**Resolved / Related Issues**
   Closes #12821 
   Closes #12947 

**Validation**
How did you test these changes?
   1. Open app and select a folder
   2. Open the preview pane
   3. Play with tags

**Screenshots (optional)**
![Screenshot 2023-07-13 234208](https://github.com/files-community/Files/assets/102259289/5f3f3c25-6619-4658-85a2-c9b671d70558)
